compiler-core/src/parse/token.rs

```diff
@@ -11,6 +11,7 @@ pub enum Token {
     Float { value: SmolStr },
     String { value: SmolStr },
     CommentDoc { content: String },
+    Attribute,
```

Comment:
Could you call this `At` and put it in the punctuation section please?

Reply:
Done!